### PR TITLE
Move ingress example into daemonset folder

### PR DESCRIPTION
**Pull request by esbie**
Makes daemonset examples more consistent. We should reorganize this folder... it's got a lot of unrelated stuff in it.

### CODE REVIEW

File: k8s-daemonset/README.md [hunk 1]
```diff
@@ -149,6 +149,15 @@ And lastly, once you have linkerd deployed, you can deploy
 kubectl apply -f https://raw.githubusercontent.com/BuoyantIO/linkerd-viz/master/k8s/linkerd-viz.yml
 ```
 
+#### Daemonsets + Ingress Controller
+
+To router to external requests using ingress resources, deploy linkerd as an ingress controller, and create an ingress resource:
```
Comment (klingerf) [inline k8s-daemonset/README.md]: Typo here -- I think it should be:

> To route external requests using ingress resources, ...

Also, tiny nit: mind wrapping at 80 characters?

**Comment by esbie** [comment]
yeah it's intentional, it's too confusing to try to differentiate `linkerd-ingress.yml` from `linkerd-ingress-controller.yml` so I changed the name. Nginx isn't referenced, but that linkerd daemonset is meant to run behind the nginx defined in `nginx.yml`

**Comment by rmars** [comment]
`linkerd-ingress.yml` is mentioned in V and VI of our blog series so we'll need to change buoyant-blog as well

**Comment by esbie** [comment]
sure happy to do that as part of this change

**Comment by esbie** [comment]
Meh, reverted the filename change after all. `linkerd-ingress` isn't ideal, but `linkerd-nginx` is a bad name too.

File: k8s-daemonset/cert.pem [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+4quXKqhfNGZvh1UubpgweZV4vRyoWhvbYo/GXmc8uO3b49iLoPvk7VHKhRKt9zvx
+j4y0bwZdrt2sSyRhKpioAEJRnc3/ow1UT81m5+IOIDg3mDPuwIcR7kprsQFfMsDK
+ONil3XWNkBNvRxmSis+ch0v94nKbktgOicD7/HGOxiXVsBNqw1OkhlNj7bj3LSbQ
+2GK4crpZppeIzzYgsoE2CeqXUf96XU3AdR3gsiep6ayM8eqa6E4OAXsDiqvJYE/i
+R3WxwoI1a6BnYX2B/31X9B5qxhTNrRU7p/DukSdXxaXgFiaImFJKUskCilhiXbyc
+eKsUgFUJAIgYZekdCcPqClwBZCak7x+3b6Pu+zBqLG/kF9mo7SL9vwa0bLe78dc+
+yvT5I0z5x8wbVVPxJttMK/sBoWMoVbfn9sJZjL42RDfCGev+TOAQ/QjyStgkZfeB
+rwf50sNwWpzH6bzUMGNU+FymoohpUPKp8hYd5oFJ5eGvXamiCCqccs5mrw==
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
```
Comment (klingerf) [inline k8s-daemonset/cert.pem]: There are actually 2 certificates in this file. I don't know enough about TLS to know if that's a good or bad thing :)

Would you mind moving this into a `certificates` sub-directory, and adding a README that tells people that these are for demo purposes only, not to be copied and used? I think that will help to keep the top-level dir a bit more streamlined.

Comment (esbie) [inline k8s-daemonset/cert.pem]: huh yeah not sure about the two certs thing, this is what I get when running https://github.com/olix0r/eg-ca/blob/master/mksvc.sh and I verified it works so :woman_shrugging: 

I like the subdir idea

**Pull request merged**